packages/langium-cli/src/generator/grammar-serializer.ts

```diff
@@ -16,7 +16,7 @@ export function serializeGrammar(services: LangiumServices, grammars: Grammar[],
     if (config.langiumInternal) {
         node.append(
             `import type { Grammar } from './ast${config.importExtension}';`, NL,
-            `import { loadGrammarFromJson } from '../../utils/grammar-util${config.importExtension}';`);
+            `import { loadGrammarFromJson } from '../../utils/grammar-loader${config.importExtension}';`);
```

Comment:
Just saying this because I was in the generator PR before: Why not using the `expandToNode` function?

Reply:
Because this code predates the `expandToNode` function by ~1 year.

Reply:
This is going to change in #1287